CoreTesting/src/au/gov/asd/tac/constellation/testing/construction/ImageGraphBuilderPlugin.java

```diff
@@ -134,127 +155,151 @@ protected void edit(final GraphWriteMethods graph, final PluginInteraction inter
+        final boolean addLayers = (boolean) parameters.getObjectValue(ADD_LAYERS_PARAMETER_ID);
+        final int stateAttributeId = LayersViewConcept.MetaAttribute.LAYERS_VIEW_STATE.ensure(graph);
+        final LayersViewState currentState = graph.getObjectValue(stateAttributeId, 0);
+        if (currentState != null) {
+            // set start layer to existing graph layer count + 1
+            layer = currentState.getLayerCount() + 1;
+        }
+        
+        for (final BufferedImage image : images) {
+
+            if (addLayers && layer > 1) {
+                // add new layer for each image if addLayer option selected
+                PluginExecution.withPlugin(new NewLayerPlugin()).executeNow(graph);
+            }
+
+            // If add right option selected, images will be appended to the
+            // right, otherwise, images will be added on top of the prev one
+            if (w > 0 && addRight) {
+                prevWidth = w + prevWidth;
             }
+            w = image.getWidth();
+            final int h = image.getHeight();
+            final int[][] vertexIds = new int[w][h];
+
+            final float zlen = multipleFrames ? 0 : Math.min(w, h) / 4F;
+            final float vis = 1;
+
+            // get layer bitmask attribute          
+            final int vertexBitmaskAttributeId = LayersConcept.VertexAttribute.LAYER_MASK.ensure(graph);
+
+            for (int x = 0; x < w; x++) {
+                for (int y = 0; y < h; y++) {
+                    final int rgb = image.getRGB(x, y);
+                    final int a = (rgb >> 24) & 0xff;
+                    final int r = (rgb >> 16) & 0xff;
+                    final int g = (rgb >> 8) & 0xff;
+                    final int b = rgb & 0xff;
+
+                    // Generate Z using grayscale.
+                    final float gray = 0;
+
+                    final int vxId = vertexIds[x][y] = graph.addVertex();
+                    // Add identification for each image
+                    graph.setStringValue(vertexIdentifierAttributeId, vxId, String.format("image%d,%d,%d", layer, x + prevWidth, y));
+
+                    final int yinv = h - y;
+                    ConstructionUtilities.setxyz(graph, vxId, vertexXAttributeId, vertexYAttributeId, vertexZAttributeId, (x + prevWidth) * 2, yinv * 2, -gray * zlen / 255F);
+                    ConstructionUtilities.setxyz(graph, vxId, vertexX2AttributeId, vertexY2AttributeId, vertexZ2AttributeId, (x + prevWidth) * 2, yinv * 2, 0);
+                    graph.setStringValue(vertexBackgroundIconAttributeId, vxId, "Background.Flat Square");
+                    final ConstellationColor color = ConstellationColor.getColorValue(r / 255F, g / 255F, b / 255F, a / 255F);
+                    graph.setObjectValue(vertexColorAttributeId, vxId, color);
+
+                    if (multipleFrames) {
+                        graph.setFloatValue(vertexVisibilityAttributeId, vxId, vis);
```

Comment:
This seems unnecessary. 
The `vis` value is always 1, which is already the default visibility value.
Should be able to remove the variables and usages of both `vertexVisibilityAttributeId` and `vis`

Reply:
Removed vertexVisibilityAttributeId and vis